pom.xml

```diff
@@ -250,6 +251,7 @@
                               <exclude>**/src/site/resources/**/*.txt</exclude>
                               <exclude>**/src/site/resources/**/*.json</exclude>
                               <exclude>**/.gitignore</exclude>
+                              <exclude>**/*.md</exclude>
```

Comment:
@steveblackmon In moving the excludes to streams-master, for exceptions and overrides, thoughts on whether they should be scoped at the level of the sub-project (streams-project, streams-examples) or the module (streams-project/streams-verbs)?